src/components/Accordion/Accordion.tsx

```diff
@@ -38,29 +41,42 @@ export function FAQ(props: IFAQ) {
   );
 }
 
-export function FAQCategory(props: IFAQCategory) {
-  const { categoryTitle, children, state } = props;
+export function AccordionCategory(props: AccordionCategory) {
+  const { categoryTitle, children, state, iconOpen, iconClose, isChild } =
+    props;
   const [active, setActive] = useState<boolean>(state || false);
 
+  const baseIconClose = <FiMinus size={20} />;
+  const baseIconOpen = <FiPlus size={20} />;
+
+  const isNotChild = isChild === false;
+
   return (
-    <div>
-      <hr
-        className="h-0.5 border-t-0 bg-transparent bg-gradient-to-r 
+    <div className="text-neutral">
+      {isNotChild ? (
+        <></>
```

Comment:
I think you can make the opposite here just using &&, isn't it!?

```suggestion
      {isChild && 
        <hr className="h-0.5 border-t-0 bg-transparent bg-gradient-to-r 
```

Reply:
if i pass directly the boolean isChild as argument of ternary and look at the case where i don't pass isChild in args it evaluates to undefined and always equals false

Reply:
the objective is to only say when necessary if is element is not child

Reply:
but i guess it would be acceptable to pass the isChild bool param everytime we use it ?